india_compliance/gst_india/report/gstr_1_document_issued_summary/gstr_1_document_issued_summary.py

```diff
@@ -0,0 +1,196 @@
+# Copyright (c) 2022, Resilient Tech and contributors
+# For license information, please see license.txt
+
+import frappe
+from frappe import _
+from frappe.model.naming import parse_naming_series
+
+
+def execute(filters=None):
+    columns = get_columns(filters)
+    data = get_data(filters)
+
+    return columns, data
+
+
+def get_columns(filters) -> list:
+    return [
+        {
+            "fieldname": "nature_of_document",
+            "label": _("Nature of Document"),
+            "fieldtype": "Data",
+            "width": 300,
+        },
+        {
+            "fieldname": "naming_series",
+            "label": _("Series"),
+            "fieldtype": "Data",
+            "width": 150,
+        },
+        {
+            "fieldname": "from_serial_no",
+            "label": _("Serial Number From"),
+            "fieldtype": "Data",
+            "width": 160,
+        },
+        {
+            "fieldname": "to_serial_no",
+            "label": _("Serial Number To"),
+            "fieldtype": "Data",
+            "width": 160,
+        },
+        {
+            "fieldname": "total_number",
+            "label": _("Total Submitted Number"),
+            "fieldtype": "Int",
+            "width": 180,
+        },
+        {
+            "fieldname": "canceled",
+            "label": _("Canceled Number"),
+            "fieldtype": "Int",
+            "width": 160,
+        },
+        {
+            "fieldname": "total_issued",
+            "label": _("Total Issued Documents"),
+            "fieldtype": "Int",
+            "width": 150,
+        },
+    ]
+
+
+def get_data(filters) -> list:
+    data = []
+
+    document_mapper = {
+        "Invoices for outward supply": "Sales Invoice",
+        "Invoices for inward supply": "Purchase Invoice",
+        "Invoices for inward supply from unregistered person": "Purchase Invoice",
+        "Debit Note": "Purchase Invoice",
+        "Credit Note": "Sales Invoice",
+        "Receipt Voucher": "Payment Entry",
+        "Payment Voucher": "Payment Entry",
```

Comment:
Did you refer to Refund Voucher here?

https://cleartax.in/s/refund-voucher-to-be-issued-for-advance-payment
https://cleartax.in/s/payment-voucher-reverse-charge-gst

If it's confusing, ignore this implementation for now, and we can add this as a feature later on.